modules/preview-react/information-highlight/stories/InformationHighlight.stories.mdx

```diff
@@ -0,0 +1,80 @@
+
```

Comment:
This will have to be updated to match our new `mdx` setup. 

I would reference what is now in `master` to match that file setup. It will be a `mdx` file and a `stories` file.

If you have any questions, don't hesitate to ask!

Reply:
done! in other pr :D